### PR TITLE
Revert "fix: hot reloading experience + relay-config removal"

### PR DESCRIPTION
**Pull request by gkartalis**
Reverts artsy/eigen#8761

### CODE REVIEW

**Comment by gkartalis** [comment]
https://github.com/artsy/eigen/pull/8761#issuecomment-1563257043

**Review by dimatretyak — APPROVED** [review]

**Comment by gkartalis** [comment]
So I am having some issues so I'm gonna revert, app seems to be working fine, query map is being generated as expected but when I go to search and search for "Banksy" I get the result and try to go to the artist page and the app crashes with the following:

simulator_screenshot_2593491E-AADE-4D32-A74D-E15AA711B90F

when I grab the queryID I can confirm that it is in the queryMap 🤷 

cc @brainbicycle

**Comment by gkartalis** [comment]
but def weird that `yarn relay` doesn't generate the query map on main 🤔

File: src/app/system/relay/middlewares/metaphysicsMiddleware.ts [hunk 1]
```diff
@@ -109,8 +110,12 @@ export function persistedQueryMiddleware(): Middleware {
     let body: { variables?: object; query?: string; documentID?: string } = {}
     const queryID = req.getID()
     const variables = req.getVariables()
-
-    body = { documentID: queryID, variables }
+    if (__DEV__) {
+      body = { query: require("../../../../../data/complete.queryMap.json")[queryID], variables }
+      ;(req as any).operation.text = body.query ?? null
```
[the remark this answers is not in the document]
Comment (brainbicycle) [reply src/app/system/relay/middlewares/metaphysicsMiddleware.ts:115]: as in, keeping the repersist flag and adding this if statement, is the red screen still occurring? We can make a ticket don't have to go too deep in the weeds

[the remark this answers is not in the document]
Comment (gkartalis) [reply src/app/system/relay/middlewares/metaphysicsMiddleware.ts:115]: removed the squash on green and checkin

[the remark this answers is not in the document]
Comment (gkartalis) [reply src/app/system/relay/middlewares/metaphysicsMiddleware.ts:115]: im super curious 🕵️

**Review by olerichter00 — APPROVED** [review]

**Pull request merged**